src/Routers/FunctionsRouter.js

```diff
@@ -26,19 +26,19 @@ export class FunctionsRouter extends PromiseRouter {
       }
     }
   }
-  
+
   static handleCloudFunction(req) {
     var applicationId = req.config.applicationId;
     var theFunction = triggers.getFunction(req.params.functionName, applicationId);
     var theValidator = triggers.getValidator(req.params.functionName, applicationId);
     if (theFunction) {
-
       const params = Object.assign({}, req.body, req.query);
       var request = {
         params: params,
         master: req.auth && req.auth.isMaster,
         user: req.auth && req.auth.user,
-        installationId: req.info.installationId
+        installationId: req.info.installationId,
+        log: req.config.loggerController && req.config.loggerController.adapter
```

Comment:
Reading is quite interesting, a lot of loggers i have looked at don't support reading at all.

For people wanting to use their own loggers, I'm not sure how important having the read call is.

I can see it being very useful for people who don't care for configuring their own logger though, so they can quickly check their logs in the dashboard (However that's currently broken and no one has said anything more in the bug i made about it #1563 ).

I would just leave the logger adapter where it is now and see if anyone asks for more. 
I personally would like a `.`trace`` but 90% of my log calls in cloud code are ``info`` or ``error``